CMakeLists.txt

```diff
@@ -2,6 +2,10 @@ cmake_minimum_required(VERSION 3.5 FATAL_ERROR)
 
 project(CMaizeSuperbuild LANGUAGES CXX)
 
+# Required to enable unit testing
+include(CTest)
+enable_testing()
```

Comment:
don't explicitly call `enable_testing()`. `include(CTest)` will call it if `BUILD_TESTING` is true